app/src/main/java/com/tristanwiley/chatse/chat/ChatActivity.kt

```diff
@@ -181,6 +185,7 @@ class ChatActivity : AppCompatActivity(), ServiceConnection {
 
         //Load a default room
         loadChatFragment(ChatRoom(prefs.getString("lastRoomSite", Client.SITE_STACK_OVERFLOW), prefs.getInt("lastRoomNum", 15)))
+        room_num = 15
```

Comment:
You can't assume that the default room for all users will be room 15. It may be true for us, but it won't be for the rest of the chatSE users.

Reply:
room_num = prefs.getInt("lastRoomNum", 15) would do for now?